### ISSUE TITLE
Window titlebars changed in Firefox 125.0.1

### ISSUE THREAD

**Issue by sekurilabs** [rafaelmardojai/firefox-gnome-theme#775]
After the last update to Firefox 125.0.1 minimize, maximize and close buttons changed and are looking awful.

![image](https://github.com/rafaelmardojai/firefox-gnome-theme/assets/1055788/1c71509a-abd8-4a9f-8623-7e23dd68930b)

**Comment by garrett** [comment]
I'm seeing this too in Firefox flatpak from Flathub on Fedora 40 beta (although distro likely doesn't influence it).

It looks like it was also reported @ https://github.com/rafaelmardojai/firefox-gnome-theme/issues/767 (when 125 was in beta).

**Comment by sekurilabs** [comment]
> I'm seeing this too in Firefox flatpak from Flathub on Fedora 40 beta (although distro likely doesn't influence it).
> 
> It looks like it was also reported @ #767 (when 125 was in beta).

Yeah you are right. It's not a big change but it's quite annoying as it reminds me of linux UI from 15 years ago.

**Comment by garrett** [comment]
It's so bad, even on default Firefox:

![Screenshot from 2024-04-17 10-48-03](https://github.com/rafaelmardojai/firefox-gnome-theme/assets/10246/ceb64f37-1785-499f-aa1a-dc74e8d4c5c5)

And the various official color themes are horrible too:

![Screenshot from 2024-04-17 10-51-17](https://github.com/rafaelmardojai/firefox-gnome-theme/assets/10246/b84efbef-da90-455d-860c-bfd8ec11c530)

It's even worse if you have more than close enabled as well. Here, maximize is being hovered and it's black on dark purple, so it's almost invisible:

![image](https://github.com/rafaelmardojai/firefox-gnome-theme/assets/10246/2e6d02a7-45a5-4021-b2e4-9becbc5bd944)

**Comment by prastowoagungwidodo** [comment]
You can temporary fix this issue by delete or renaming files:
- window-close-symbolic.svg
- window-maximize-symbolic.svg
- window-minimize-symbolic.svg
- window-restore-symbolic.svg
inside "<profile-path>/chrome/firefox-gnome-theme/theme/icons" directory

**Comment by sekurilabs** [comment]
I've also started creating a PPA for Ubuntu (and derivates) users so they (you) can install the theme from PPA. I plan to maintain it for future releases. Currently working on v124.

**Comment by sekurilabs** [comment]
> You can temporary fix this issue by delete or renaming files:
> 
>     * window-close-symbolic.svg
> 
>     * window-maximize-symbolic.svg
> 
>     * window-minimize-symbolic.svg
> 
>     * window-restore-symbolic.svg
>       inside "/chrome/firefox-gnome-theme/theme/icons" directory

This removes the tab close button icon and leaves it empty.

**Comment by prastowoagungwidodo** [comment]
You can add/edit customChrome.css (chrome/firefox-gnome-theme/customChrome.css) file and add the following code:

`
.tabbrowser-tab .tab-close-button {
 list-style-image: url("moz-icon://stock/window-close-symbolic?size=dialog") !important;
}
`

**Comment by rafaelmardojai** [comment]
Duplicated of #767.
